db/schema.rb

```diff
@@ -56,6 +57,7 @@
     t.string   "question"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.integer  "order"
```

Comment:
The schema file is generated from the migrations. The reason why we keep it in source control is because if you want to deploy your application on a new machine you'd have to run all of your migrations and if your application is a few years old you could have hundreds of migrations so it makes more sense to just upload the latest file.